qgreenland/util/luigi/tasks/main.py

```diff
@@ -106,28 +107,37 @@ def run(self):
             )
 
 
+class LinkLayer(QgrLayerTask):
+
+    def output(self):
+        return luigi.LocalTarget(
+            get_layer_compile_dir(self.node),
+        )
+
+    def run(self):
+        with temporary_path_dir(self.output()) as temp_path:
+            for inp in Path(self.input().path).glob('*'):
+                # Hard link final layer files to the zip compile directory.
+                #
+                # NOTE: Hardlink API is backwards to the symlink API...
+                # https://docs.python.org/3/library/pathlib.html#pathlib.Path.link_to
+                inp.link_to(temp_path / inp.name)
+
+
 class FinalizeTask(QgrLayerTask):
     """Allow top-level layer tasks to lookup config from class attr layer_id.
```

Comment:
I think this docstring needs updated? We do allow top-level layer tasks to lookup config via layer_id, but we actually moved the `node` property to `QgrLayerTask`.

Reply:
1215e61

Reply:
24a7e30